qt/applications/workbench/workbench/plotting/toolbar.py

```diff
@@ -215,7 +215,7 @@ def set_buttons_visiblity(self, fig):
                     if ax.get_artists_sample_log_plot_details(artist) is not None:
                         self._set_fit_enabled(False)
                         break
-                except ValueError:
+                except Exception:
```

Comment:
I know it may be neccessary in this instance to catch all exceptions, but would it be useful to output the error somewhere (even if it's just debug) just in case something unexpected breaks in future?

Reply:
In this case no,  If there is a failure here, we can safely ignore it, Exception was picked as it will not catch any of the BaseException types that we should not catch.  I considered (ValueError, SpecificErrorThatICantRemember), but Martyn pointed out that any error that occurred there should be handled the same way.